src/braket/circuits/gate.py

```diff
@@ -40,6 +43,27 @@ def __init__(self, qubit_count: Optional[int], ascii_symbols: Sequence[str]):
                 `ascii_symbols` length != `qubit_count`
         """
         super().__init__(qubit_count=qubit_count, ascii_symbols=ascii_symbols)
+        self._adjoint = False
+
+    def adjoint(self) -> Gate:
+        """Returns the adjoint of this gate.
+
+        Returns:
+            Gate: The adjoint of this gate.
+        """
+        new = copy.copy(self)
+        new._adjoint = not self._adjoint
+        return new
+
+    def adjoint_list(self) -> List[Gate]:
```

Comment:
Isn't it weird to have both adjoint and adjoint_list? When would we know which one to use? Why not just always use adjoint_list?

Reply:
All `adjoint` does is mark a gate as an adjoint; it doesn't have anything to do with the actual implementation of adjoints. `adjoint_list`, on the other had, is a definition of the adjoint. In principle, `adjoint_list` isn't necessary at all; the adjoint implementation could be handled by the device, but we currently have no way of communicating that.

Reply:
Could we differentiate the names more to make this difference more obvious?

Reply:
`adjoint_expansion`?